run_in_kubernetes.sh

```diff
@@ -11,9 +11,9 @@ kubectl run sampleloader -it --rm \
    --image $IMAGE\
    --env=RABBITMQ_USER=$(kubectl get secret rabbitmq -o=jsonpath="{.data.rabbitmq-username}" | base64 --decode) \
    --env=RABBITMQ_PASSWORD=$(kubectl get secret rabbitmq -o=jsonpath="{.data.rabbitmq-password}" | base64 --decode) \
-   --env=RABBITMQ_SERVICE_HOST=rabbitmq \
+   --env=RABBITMQ_SERVICE_HOST=rm-rabbitmq-client \
```

Comment:
this is an extremely odd name for the hostname of the Rabbit service. _We_ are the client and Rabbit is the host

Reply:
I think this is plain wrong. It's `rm-rabbitmq` everywhere else

Reply:
Good point - that is probably why the operator people decided to change it and remove the client bit.